x-pack/packages/security/plugin_types_common/src/api_keys/api_key.ts

```diff
@@ -119,6 +119,7 @@ interface BaseQueryApiKeyResult {
 /**
  * Interface representing a REST API key that is managed by Kibana.
  */
+// @ts-expect-error `type` can only be "rest" or "cross_cluster"
```

Comment:
@JoshMock

Reply:
[PR in progress](https://github.com/elastic/kibana/pull/205174)